registry/storage/layerwriter.go

```diff
@@ -214,6 +214,7 @@ func (lw *layerWriter) resumeHashAt(offset int64) error {
 		return fmt.Errorf("unable to get stored hash states with offset %d: %s", offset, err)
 	}
 
+	ctx := lw.layerStore.repository.ctx
```

Comment:
Let's put a note here that this is not the ideal context. It should come from the method call above.